src/Corax/Indexing/IndexWriter.cs

```diff
@@ -131,7 +130,6 @@ private void Init()
             _entryIdToLocation = _transaction.LookupFor<Int64LookupKey>(Constants.IndexWriter.EntryIdToLocationSlice);
             _jsonOperationContext = JsonOperationContext.ShortTermSingleUse();
             _fieldsTree = _transaction.CreateTree(Constants.IndexWriter.FieldsSlice);
-            _persistedDynamicFieldsAnalyzers = _transaction.CreateTree(Constants.IndexWriter.DynamicFieldsAnalyzersSlice);
```

Comment:
I've removed `_persistedDynamicFieldsAnalyzers` tree since we're not longer needs it. Previously (I mean v5.4) we were storing original term and during deletion we had to analyze it again to retrieve correct term (in meaning the one which is in the `CompactTree`) so we had to store configuration in tree since it may varied for each document. Since we introduced "EntryTable" which contains terms IDs we no longer have original term and deletion happens by term id there is no need to persist it. 

Am I correct? cc @ayende @redknightlois

Reply:
Sounds about correct, yes.